public/index.html

```diff
@@ -76,12 +76,12 @@ <h1>
 			const output = document.getElementById('output');
 
 			/**
-			 * Figures out if a given url is a webhook
+			 * Figures out if a given url is a valid Discord webhook
 			 * @param url {string} The webhook URL
-			 * @return url is `https://discord.com/api/webhooks/${string}/${string}`
+			 * @return a boolean indicating if the url is `https://discord.com/api/webhooks/${string}/${string}`, `https://canary.discord.com/api/webhooks/${string}/${string}`, or `https://ptb.discord.com/api/webhooks/${string}/${string}`
 			 */
 			function isWebhook(url) {
-				return url.startsWith('https://discord.com/api/webhooks/');
+				return ["https://discord.com", "https://canary.discord.com", "https://ptb.discord.com"].some(domain => url.startsWith(domain) && url.includes('/api/webhooks'));
```

Comment:
The discord client application whitelists *.discord.com, and just cuts it off!
IE: you can test yourself at https://andr222ew.discord.com/channels/720454859974770769/720454860566167574/965788644147331083
(note that this is only for clicking links in the app, not acual domains!)
Therefore It would make sense (and to ensure future compatibility with other possible test domains) that you use a regex to cut it off!